### ISSUE TITLE
Subpleural curvilinear line

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6712]
**Preferred term label:**
Subpleural curvilinear line

**Synonyms**

**Definition (free text, please give PubMed ID)**
This finding is a thin curvilinear opacity, 1-3 mm in thickness, lying less than 1 cm from and parallel to the pleural surface (Fig 68). It corresponds to atelectasis of normal lung if seen in the dependent posteroinferior portion of lung of a patient in the supine
position and is subsequently shown to disappear on CT sections acquired with the patient prone. It may also be encountered in patients with pulmonary edema or fibrosis (other signs are usually present). 

PMID:18195376

Comment
Though described in the context of asbestosis, subpleural curvilinear line is not specific for asbestosis. See Figure 68 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
